docs/content/en/docs/install/tips-tricks.md

```diff
@@ -0,0 +1,152 @@
+---
+title: Installation Tips and Tricks
+description: Supplemental installation information
+weight: 38
+hidechildren: false # this flag hides all sub-pages in the sidebar-multicard.html
+--- 
+
+The
+[Install Keptn](install.md)
+page documents how to install Keptn.
+This page provides some background and more examples
+that supplement that information.
+
+## Umbrella chart
+
+Keptn v0.9.0 and later is installed using a Helm umbrella chart.
+This means that the Helm Chart that installs all of Keptn
+actually groups subcharts for individual components
+and you can install individual components
+without installing all of Keptn.
+
+Keptn is installed using the
+[keptn/values.yaml](https://github.com/keptn/lifecycle-toolkit-charts/blob/main/charts/keptn/values.yaml)
+umbrella chart.
+Each subchart has its own README file describing possible configuration options,
+but configuration changes for the subcharts are added to a single `values.yaml` file.
+See
+[Customizing the configuration of components](install.md/#customizing-the-configuration-of-components)
+for an example.
+
+## Installing older versions of Keptn
+
+Installation of Keptn v0.9.0 has two significant differences
+compared to the installation of earlier releases:
+
+* Keptn v0.9.0 and later releases use
+  the umbrella charts whereas earlier versions did not
+* Keptn v0.9.0 and later releases use
+  `keptn` as the value to the `NAME` option of the
+  [helm repo add](https://helm.sh/docs/helm/helm_repo_add/)
+  command whereas earlier releases used `klt`.
+
+To install a version prior to v0.9.0,
+use the install command sequence that is documented for that release.
+To install the latest version use:
+  
+```shell
+helm repo add keptn https://charts.lifecycle.keptn.sh
+helm repo update
+helm search repo keptn
+```
```

Comment:
Let's not repeat more commands here. We could just link back to the install page.
```suggestion
To install the latest version, use the installation commands on the
[Install Keptn](install.md) page.
```